Compliance.md

```diff
@@ -30,3 +32,38 @@ The [BSI TR-03183-2 v1.1](https://www.bsi.bund.de/SharedDocs/Downloads/EN/BSI/Pu
 | | `URI of the executable form`| component->externalReferences->type (distribution/distribution-intake) | PackageDownloadLocation | |
 | | `hash of source code`| no-deterministic-field | package->PackageVerificationCode | |
 | | `other uniq identifiers`| component->cpe, component->purl| package->externalReference->security (cpe/purl) | |
+
+## OpenChain Telco: SBOM Requirements for OCT
+
+The [OpenChain Telco](https://github.com/OpenChain-Project/Reference-Material/blob/master/SBOM-Quality/Version-1/OpenChain-Telco-SBOM-Guide_EN.md) specifies mandatory properties for an SBOM. Below is how we have derived all the values.
+| OpenTelco | Section ID | OpenTelco field | SPDX(2.3) | Notes |
```

Comment:
Remove all the details in () not required. e.g DataLicense(SBOM_LICENSE) we don't require (SBOM_LICENSE)